radio/src/targets/horus/tp_gt911.cpp

```diff
@@ -216,13 +216,13 @@ const uint8_t TOUCH_GT911_Cfg[] = {
 //GT911 param table
 const uint8_t TOUCH_GT911_Cfg[] =
   {
-    GT911_CFG_NUMER,     // 0x8047 Config version
+    GT911_CFG_NUMBER,    // 0x8047 Config version
     0xE0,                // 0x8048 X output map : x 480
     0x01,
     0x10,                // 0x804A Y ouptut max : y 272
     0x01,
-    GT911_MAX_TP,        // 0x804C Touch number
-    0x3C,                // 0x804D Module switch 1 : bit4= xy change Int mode
+    GT911_MAX_TP,        // 0x804C Touch number 
+    0x3C, // 0x804D Module switch 1 : bit4= xy change Int mode    
```

Comment:
Minor: can you please fix the indentation (whitespace) on this line.

Reply:
Done: but I left the T18 mess alone